04/nicdard.ml

```diff
@@ -23,6 +23,14 @@ let easy_tests =
   ; ("or2.oat", "", "26")
   ; ("sgstring.oat", "", "0")
   ; ("nullref.oat", "", "0")
+  ; ("shoisted.oat", "", "0")
```

Comment:
The function Gradedtest.executed_oat_file will return the standard output and the return code concatinated, thus this test will return "local string0".

Reply:
Thanks! I wrote the test before implementing function calls, that's why I didn't try it. Now it should be fixed :)